wiki/configuration.md

```diff
@@ -0,0 +1,17 @@
+<div id="main_content_wrap" class="outer">
+
+## Configuration
+
+This project does not need any configuration. It is used as external library that will be included as a dependency to other projects.
```

Comment:
Perhaps add a link to the usage of semantics here?

Reply:
added usage link